examples/basic-examples/pages/CustomIcon.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div class="custom-icon">
+    <p>
+      Adding a new icon not defined in Ripple:
+      <rpl-icon symbol="lock_open" color="primary" size="xl" />
+    </p>
+    <p>
+      Replacing Ripple "search" icon with different design:
+      <rpl-icon symbol="search" color="secondary" size="xl" />
+    </p>
+  </div>
+</template>
+
+<script>
+import RplIcon from '../../../packages/components/Atoms/Icon/index.js'
+// Imports below will be moved to ripple-nux-ui so we can enable/disable customIcon.
+import './../assets/ripple-icon/custom_icon_lock_open'
+import './../assets/ripple-icon/custom_icon_search'
```

Comment:
@tim-yao we just need to import custom icons like this. Below is my test page.
 ![image](https://user-images.githubusercontent.com/14160054/95399882-f65b1800-0954-11eb-8f24-081a8fb205cb.png)